.bcr/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+matrix:
+  platform: ["debian10", "macos", "ubuntu2004"]
```

Comment:
Can we add windows here too?

Reply:
yes, updated to match the content of the last time the module was published: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/protobuf/23.1/presubmit.yml